cmd/list.go

```diff
@@ -9,13 +9,16 @@ import (
 
 // NewListCmd returns list subcmd
 func NewListCmd(globalCfg *config.GlobalImpl) *cobra.Command {
+	var skipCharts bool
 	listOptions := config.NewListOptions()
 	listImpl := config.NewListImpl(globalCfg, listOptions)
 
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List releases defined in state file",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			listOptions.WithPreparedCharts = !skipCharts
```

Comment:
why not listOptions.SkipCharts?

Reply:
ping @stek29

Reply:
Currently SkipChart is only used as a separate parameter of the List subcommand, which does not affect the other logic. I think using SkipChart will make the code style more consistent and unified. @stek29

Reply:
@yxxhero ok, i'll update to SkipCharts